.eslintignore

```diff
@@ -22,3 +22,6 @@ packages/cli/install/
 packages/cli/preinstall/
 packages/cli/**/tmp-*
 reproductions/
+
+
+node_modules/
```

Comment:
Not needed as already ignored by default https://eslint.org/docs/latest/use/configure/ignore